### ISSUE TITLE
why the ddpm can't generate images well? 

### ISSUE THREAD

**Issue by awais00012** [explainingai-code/StableDiffusion-PyTorch#3]
i train the ddpm for 200 epochs and the result is very poor

**Comment by explainingai-code** [comment]
Hello @awais00012 , I would need some more information to help with this.
1. When you say the results are poor, is it the latent image generation or the decoded image that is poor.
2. Do the loss/generated samples get any better through the training.
3. Were auto encoder results satisfactory or even that were bad ?
4. Was this a a conditional ldm or unconditional. Could you add the config here.
5. Also could you provide a sample dataset image, encoded latent and reconstructed version of this image by autoencoder, and any pair of sample of generated latent and decoded image.

**Comment by awais00012** [comment]
The auto encoder results are awesome. the unconditional ldm result are not good, the losses is also not better during the training. also i add the evaluation metrics to the code and i observed that the model  give psnr ratio and ssim in negative and was not stable. 
note: i run the model a 10 days ago i don't know that whether you updated the repo or not. i have the unconditional repo
